db/models.py

```diff
@@ -1 +1,44 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
```

Comment:
Consider whether the 'bonus' field in the Skill model should be a CharField. If it represents a numerical value, a different field type like IntegerField or FloatField might be more appropriate.